### PR TITLE
Add Mike Lippincott

### PR DESCRIPTION
**Pull request by MikeLippincott**
Add Mike Lippincott to WayScience Lab website. @gwaybio

### CODE REVIEW

File: _members/mike-lippincott.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+name: Mike Lippincott
+image: images/team/\
```
Comment (gwaybio) [inline _members/mike-lippincott.md:3]:
```suggestion
image: images/team/mike_lippincott.jpg
```

File: _members/mike-lippincott.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    - lippincm
+links:
+  orcid: 0000-0002-8637-1448
+  home-page: https://github.com/MikeLippincott
+  email: michael.lippincott@cuanschutz.com
+  github: MikeLippincott
+  twitter: mike_lippincott
+---
+
+Mike Lippincott is a Cell Biology, Stem Cells, and Development (CSD) PhD student at the University of Colorado, Anschutz Medical Campus. Mike joined the Way Lab in July 2023 and is looking to understand how cellular morphologies change in the context of cellular stress and how organelle interactions affect cellular states.
```
Comment (gwaybio) [inline _members/mike-lippincott.md:20]:
```suggestion
Mike Lippincott is a Cell Biology, Stem Cells, and Development (CSD) PhD student at the University of Colorado, Anschutz Medical Campus.
Mike joined the Way Lab in July 2023 and is looking to understand how cellular morphologies change in the context of cellular stress and how organelle interactions affect cellular states.
```

Comment (gwaybio) [reply _members/mike-lippincott.md:20]: Can you make sure all is one sentence per line?

File: _members/mike-lippincott.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+Mike earned a B.S. in Biochemistry and Data Science from [Maryville University of Saint Louis](https://www.maryville.edu/) in 2021. Here he conducted research in biochemical pathway manipulations in sugar cane, CRISPR gene editing in non-mendelian genetic plant models, plasma based atomic emission spectroscopy, and cellular transcriptomic investigation of telomerase activating drug TA-65.
+
+After graduating from [Maryville](https://www.maryville.edu/), Mike did his post-baccalaureate training at the [University of North Caroina at Chapel Hill](https://www.unc.edu/) where he developed software to analyze telomeric sequence homology in [Caenorabditis elegans](http://www.wormbook.org/) and Humans. 
+
+Mike is passionate about collaborative team science and bridging the gap between scientific fields to promote interdisciplinary science.
```
Comment (gwaybio) [inline _members/mike-lippincott.md:26]: love it! Can you add your _why_ statement? Why are you passionate? (it currently reads as a _what_ statement, which is less powerful)

File: _members/mike-lippincott.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+Mike earned a B.S. in Biochemistry and Data Science from [Maryville University of Saint Louis](https://www.maryville.edu/) in 2021. Here he conducted research in biochemical pathway manipulations in sugar cane, CRISPR gene editing in non-mendelian genetic plant models, plasma based atomic emission spectroscopy, and cellular transcriptomic investigation of telomerase activating drug TA-65.
+
+After graduating from [Maryville](https://www.maryville.edu/), Mike did his post-baccalaureate training at the [University of North Caroina at Chapel Hill](https://www.unc.edu/) where he developed software to analyze telomeric sequence homology in [Caenorabditis elegans](http://www.wormbook.org/) and Humans. 
+
+Mike is passionate about collaborative team science and bridging the gap between scientific fields to promote interdisciplinary science. 
+
+Outside of lab you can find Mike rock climbing, snowboarding, 3D printing, photographing nature, or tending to his aquatic pets the Neocaridina davidi.
```
Comment (gwaybio) [inline _members/mike-lippincott.md:28]:
```suggestion
Outside of lab you can find Mike rock climbing, snowboarding, 3D printing, photographing nature, or tending to his aquatic pets the _Neocaridina davidi_.
```

**Review by gwaybio — APPROVED** [review]
LGTM

Minor items only. You may also need to resize your headshot jpg, which we should preview in the netlify deploy preview

**Comment by MikeLippincott** [comment]
@gwaybio I addressed comments and added a why statement. We can address the Netlify later in the week. Have a happy 4th!

File: _members/mike-lippincott.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
+---
+name: Mike Lippincott
+image: images/team/mike_lippincott.jpg
```
Comment (gwaybio) [inline _members/mike-lippincott.md:3]:
```suggestion
image: images/team/mike_lippincott.jpeg
```

**Pull request merged**